doc/source/dev/testing.txt

```diff
@@ -18,3 +18,25 @@ Example::
 
     $ cd examples/errors
     $ GNUPGHOME=/home/user1/clones/batou/src/batou/secrets/tests/fixture/gnupg ./batou deploy errors
+
```

Comment:
I think this wants to be in the original merge, not the specific migration step for the environments/secrets.

Reply:
We put it here because we needed the knowledge of writing an actual migration step to find out about it.
As the original PR is already merged, I'd like to keep it here.